.evergreen/legacy_config_generator/evergreen_config_lib/variants.py

```diff
@@ -53,18 +53,18 @@ def days(n: int) -> int:
             "link-with-cmake-ssl-deprecated",
             "link-with-cmake-snappy",
             "link-with-cmake-snappy-deprecated",
-            OD([("name", "link-with-cmake-mac"), ("distros", ["macos-1014"])]),
-            OD([("name", "link-with-cmake-mac-deprecated"), ("distros", ["macos-1014"])]),
+            OD([("name", "link-with-cmake-mac"), ("distros", ["macos-1100"])]),
```

Comment:
Update or remove references to `macos-1014` variants found in other files. [Example](https://github.com/mongodb/mongo-c-driver/blob/0e54f99dada9db1196f82a7ef868f58ca8b0f022/.evergreen/config_generator/components/sasl/darwinssl.py#L18-L19).

Update or remove references to `macos-1015` found in other files. CDRIVER-4580 proposes dropping macOS 10.15 too.

Update [platforms.rst](https://github.com/mongodb/mongo-c-driver/blob/master/src/libmongoc/doc/ref/platforms.rst?plain=1#L33) to remove reference to macOS 10.14.

Reply:
Wow, I was so focused on the main Evergreen config that I missed the other occurrences. Change inbound.